browser/remoteregistry.go

```diff
@@ -0,0 +1,18 @@
+package browser
+
+import "github.com/grafana/xk6-browser/env"
+
+type remoteRegistry struct {
+	isRemoteBrowser bool
+	wsURL           string
+}
+
+func newRemoteRegistry(envLookup env.LookupFunc) *remoteRegistry {
+	r := &remoteRegistry{}
+	r.wsURL, r.isRemoteBrowser = env.IsRemoteBrowser(envLookup)
```

Comment:
I've left us relying on `env`.

I would like to repurpose the `env` package to a `remoteRegistry` package, where this file could be merged with the existing `env.go` file. This would help encapsulate `remoteRegistry` so that we only expose `IsRemoteBrowser`, and allow it to retrieve and process the env var without leaking the unexported logic. At the moment anything in the `browser` pkg can access the unexported fields of the `remoteRegistry` struct. WDYT?

Reply:
I think that can make sense yes :+1:

Reply:
Resolved in 0a49751d5998dea4efb9958013c91f15fd76a37b, 670d249561b60eed255174080782a5ab28da6151 and df397e5ba2dee445b06b454408526175b26facc7